tripleblindmarket/covid-safe-paths
# Pe runtime feature flags

#### Description:

Implements runtime feature flags.

#### Screenshots:

![Simulator Screen Shot - iPhone 11 - 2020-05-03 at 17 40 53](https://user-images.githubusercontent.com/20157849/80927720-45c56880-8d65-11ea-9d8a-c3b1a131ecc4.png)

#### How to test:

Toggle the `Exposure mode` and `google_imports` flags and verify that the respective features are toggled on/off in the UI.
